quickstarts/quickstart-template.md

```diff
@@ -0,0 +1,81 @@
+ 
+
+# Title
+
+:information_source: Before using this template, read the accompanying [quickstart template guide](quickstart-template-guide.md).
+
+## Overview
+
+This quickstart guides you through:
+- [Part 1](#part-1-taskname)
+- [Part 2](#part-2-taskname)
+- [Part n](#part-n-taskname)
+
+:information_source: (Optional) Link each part to its corresponding section for easy access.
+
+It is intended for {audience}. It assumes that you have basic knowledge of: 
+- Concept 1
+- Concept 2
+- Concept 3 
+
+:information_source: Link each concept to its corresponding content that helps users understand the concepts and ideas behind your software. 
+
+## Before you begin
+
+:information_source: This section is optional.
+
+Before running this quickstart, complete the following prerequisites:
+
+- Prerequisite 1
+- Prerequisite 2
+- Prerequisite 3
+ 
+## Install
+
+:information_source: This section is optional. Use this section to provide installation instructions.
```

Comment:
**Question:** What is information source?

Reply:
https://github.com/thegooddocsproject/templates/pull/232#discussion_r827565704